ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/radiobutton/v1/radiobutton/_cq_template.xml

```diff
@@ -3,6 +3,7 @@
     jcr:primaryType="nt:unstructured"
     jcr:title="Radio Button"
     fieldType="radio-group"
+    type="number"
```

Comment:
Any reasons for adding type to OOTB component ?

Reply:
In enum we are providing `enum="[0,1]` , this is  similar to checkbox group but there we have mentioned `number` type [1]
[1]: https://github.com/adobe/aem-core-forms-components/blob/9e45e752e17abe5bde21a21ecdf64d24554e558f/ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/checkboxgroup/v1/checkboxgroup/_cq_template.xml#L6